deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/index-level-shard-allocation.md

```diff
@@ -3,27 +3,147 @@ mapped_urls:
+
+### Enabling index-level shard allocation filtering [index-allocation-filters]
+
+To filter based on a custom node attribute:
+
+1. Specify the filter characteristics with a custom node attribute in each node’s `elasticsearch.yml` configuration file. For example, if you have `small`, `medium`, and `big` nodes, you could add a `size` attribute to filter based on node size.
+
+    ```yaml
+    node.attr.size: medium
+    ```
+
+    You can also set custom attributes when you start a node:
+
+    ```sh
+    ./bin/elasticsearch -Enode.attr.size=medium
+    ```
+
+2. Add a routing allocation filter to the index. The `index.routing.allocation` settings support three types of filters: `include`, `exclude`, and `require`. For example, to tell {{es}} to allocate shards from the `test` index to either `big` or `medium` nodes, use `index.routing.allocation.include`:
+
+    ```console
+    PUT test/_settings
+    {
+      "index.routing.allocation.include.size": "big,medium"
+    }
+    ```
+
+    If you specify multiple filters the following conditions must be satisfied simultaneously by a node in order for shards to be relocated to it:
+
+    * If any `require` type conditions are specified, all of them must be satisfied
+    * If any `exclude` type conditions are specified, none of them may be satisfied
+    * If any `include` type conditions are specified, at least one of them must be satisfied
+
+    For example, to move the `test` index to `big` nodes in `rack1`, you could specify:
+
+    ```console
+    PUT test/_settings
+    {
+      "index.routing.allocation.require.size": "big",
+      "index.routing.allocation.require.rack": "rack1"
+    }
+    ```
+
+### Index allocation filter settings [index-allocation-settings]
+
+| Setting | Description |
+|---|---|
+|`index.routing.allocation.include.{{attribute}}`| Assign the index to a node whose `{{attribute}}` has at least one of the comma-separated values.|
+|`index.routing.allocation.require.{{attribute}}`| Assign the index to a node whose `{{attribute}}` has *all* of the comma-separated values.|
+|`index.routing.allocation.exclude.{{attribute}}`| Assign the index to a node whose `{{attribute}}` has *none* of the comma-separated values. |
+
+The index allocation settings support the following built-in attributes:
+
+| Attribute | Description|
+| --- | --- |
+|`_name`| Match nodes by node name |
+|`_host_ip`| Match nodes by host IP address (IP associated with hostname) |
+|`_publish_ip`| Match nodes by publish IP address |
+|`_ip`| Match either `_host_ip` or `_publish_ip` |
+| `_host`| Match nodes by hostname |
+|`_id`| Match nodes by node id |
+|`_tier`| Match nodes by the node’s [data tier](../../../manage-data/lifecycle/data-tiers.md) role. For more details see [data tier allocation filtering](elasticsearch://reference/elasticsearch/index-settings/data-tier-allocation-settings.md) |
```

Comment:
Think this is it: https://github.com/elastic/elasticsearch/blob/main/docs/reference/elasticsearch/index-settings/data-tier-allocation.md
```suggestion
|`_tier`| Match nodes by the node’s [data tier](../../../manage-data/lifecycle/data-tiers.md) role. For more details see [data tier allocation filtering](elasticsearch://reference/elasticsearch/index-settings/data-tier-allocation.md) |
```